app/models/indicator.rb

```diff
@@ -0,0 +1,78 @@
+class Indicator < ApplicationRecord
```

Comment:
I absolutely love what you've done with this class. It's far more clean and more readable.